CMakeLists.txt

```diff
@@ -949,13 +949,13 @@ if(BUILD_TESTS)
     target_link_libraries(
       historical_queries_test PRIVATE http_parser.host sss.host ccf_kv.host
     )
-
-    # Temporarily disabled flaky test
-    # https://github.com/microsoft/CCF/issues/4403 add_unit_test( indexing_test
-    # ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/indexing.cpp
-    # ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/lfs.cpp )
-    # target_link_libraries( indexing_test PRIVATE ccf_endpoints.host sss.host
-    # ccf_kv.host )
+    add_unit_test(
+      indexing_test ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/indexing.cpp
+      ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/lfs.cpp
+    )
+    target_link_libraries(
+      indexing_test PRIVATE ccf_endpoints.host sss.host ccf_kv.host
+    )
```

Comment:
I think we should temporarily enable this unit test to confirm that the leak is fixed (ie - run it through SAN), but should not re-enable when this PR is merged - the test presumably remains flaky. Whether it should also be disabled on the release branches is a separate question, but it looks like it should if we don't have time to resolve the flakiness.